### PR TITLE
Dbus doc methods

### PR DESCRIPTION
**Pull request by jremmet**
Add methods Info, Mark and GetSlotStatus

### CODE REVIEW

File: docs/reference.rst [hunk 1]
```diff
@@ -474,6 +480,69 @@ Triggers the installation of a bundle.
...
+
+The Info() Method
+^^^^^^^^^^^^^^^^^
+
+.. code::
+
+  de.pengutronix.rauc.Installer.Info()
+  Info (IN  s bundle, s compatible, s version);
+
+Triggers the installation of a bundle.
```
Comment (jluebbe) [inline docs/reference.rst]: This looks like a leftover from copying the Install() method.

**Comment by jremmet** [comment]
Thanks for the review, copy pasted mistake fixed.

**Comment by ejoerns** [comment]
Good work!

Just one question about the `Info()` method that we intentionally left undocumented because we were unsure if it is the right way to go or if we should have made a string-variant-dict-based interface for this, too.
Thus the question if you actually use the Info method and, if, how that fits your needs?

**Comment by jremmet** [comment]
Our customer asks about a D-BUS Interface to get the "rauc status" information. So I looked at  GetSlotStatus. There was no need for the Info method now. But a  string-variant-dict-based interface would allow more flexibility in future. So +1

**Comment by ejoerns** [comment]
I assume you chose the wrong button for answering?

**Comment by ejoerns** [comment]
I've started #248 as a discussion base for further D-Bus enhancements and will merge this one for now.

**Pull request merged**